frontend/src/components/Start.js

```diff
@@ -0,0 +1,229 @@
+import styled from 'styled-components/macro'
+import startscreenJpg from './../assets/startscreen.jpg'
+import wandergoldSvg from './../assets/wandergold.svg'
+import closeSvg from './../assets/close.svg'
+import compassSvg from './../assets/compass.svg'
+import { useState, useEffect } from 'react'
+
+export default function Start() {
+  const [isSearchFocused, setSearchFocus] = useState(false)
+  const [coordsToSearch, setCoordsToSearch] = useState({
+    latitude: null,
+    longitude: null,
+    locationName: '',
+    googlePlaceId: '',
+  })
+  const [suggestionList, setSuggestionList] = useState([])
+
+  useEffect(() => console.log(coordsToSearch))
+
+  return (
+    <Wrapper>
+      <LogoArea onClick={() => setSearchFocus(false)}>
+        <h1>
+          <img src={wandergoldSvg} alt="wandergold" />
+        </h1>
+      </LogoArea>
+      <LocationSearch>
+        <SearchField className={isSearchFocused && 'active'}>
```

Comment:
warum && ?
kein logischer operator?

Reply:
@web-devil it's a common practice to use logical operators for inline if. See https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator